frontend/views/containers/sidebar/GroupsMinIncome.vue

```diff
@@ -7,7 +7,7 @@ div
 
   i18n.link(
     tag='button'
-    aria-label="Change minIncome"
+    aria-label='L("Change minIncome")'
```

Comment:
This requires the `:` in front of `aria-label`, and can you please change it to read, "Change Mincome"?

Reply:
Fixed. Damn @taoeffect I'm sorry for these missing details

Reply:
No worries!